api/instance/v1/instance_sdk.go

```diff
@@ -1220,17 +1220,17 @@ type VolumeSummary struct {
 
 // VolumeTemplate volume template
 type VolumeTemplate struct {
-	// ID the volumes unique ID
+	// ID uUID of the volume
 	ID string `json:"id,omitempty"`
-	// Name the volumes name
+	// Name name of the volume
```

Comment:
double name

Reply:
I open an internal ticket to fix the documentation generation.